tests/installation/bootloader_svirt.pm

```diff
@@ -79,19 +79,55 @@ sub run() {
     # In JeOS we have the disk, we just need to deploy it, for the rest
     # - installs from network and ISO media - we have to create it.
     if (my $hddfile = get_var('HDD_1')) {
-        $svirt->add_disk({size => $size_i . 'G', file => ($vmm_family eq 'vmware') ? basename($hddfile) : $hddfile});
+        $svirt->add_disk(
+            {
+                size      => $size_i . 'G',
+                file      => ($vmm_family eq 'vmware') ? basename($hddfile) : $hddfile,
+                dev_type  => 'a',
```

Comment:
Perhaps dev_type should be called dev_name_postfix, dev_id or something similar? The device type is 'hd' for a hard disk and the 'a' identifies a particular hard disk.